Cargo.toml

```
@@ -12,11 +12,13 @@ spantrace = ["error-stack/spantrace"]
 eyre = ["error-stack/eyre"]
 serde = ["error-stack/serde"]
 hooks = ["error-stack/hooks"]
-
+grpc=[]
```

This adjusts the feature flag to only bring in the following dependencies if the feature flag is enabled.
https://doc.rust-lang.org/cargo/reference/features.html#the-features-section

```suggestion
grpc=["tonic", "tonic-types"]
```